src/views/startup_submission.hbs

```diff
@@ -36,8 +36,15 @@
   </fieldset>
 
```

Comment:
so i just checked its not an extra space its just i made a 1 line difference for each fieldset so it can look more readable